checker/tests/nullness/Issue1949.java

```diff
@@ -0,0 +1,22 @@
+import java.util.ArrayList;
+import java.util.List;
+import org.checkerframework.checker.nullness.qual.*;
+
+public class Issue1949 {
+    public interface Base<R> {}
+    //public interface Child<R> extends Base<@Nullable R>{}
```

Comment:
Instead of the commented out lines I think it would be better to explicitly have all the intended combinations.
At the moment the code doesn't correspond to the original issue, so it is hard to see what the comments mean.